### DISCUSSION TITLE
Issue #92 Headless / Command Line Operation Documentation

### DISCUSSION THREAD

**Discussion by Cortexian** [DSheirer/sdrtrunk#2093] [General]
Perhaps I am blind but I cannot locate any information on how to run SDRTrunk headless. I would like to setup some barebones remote SDR stations on cheap hardware. Miking all the performance possible out of the machine would be ideal, and that means no waterfall or GUI.

Is there any information on Issue #92 and it's 'completed' implementation? How does one go about running this headless?

**Comment by Cortexian** [comment]
@DSheirer any insight on this?

**Comment by kernworks** [comment]
All you need to do is add the following as a startup parameter:

-Djava.awt.headless=true

If you are running on windows and using the sdr-trunk.bat file for startup you can add this to the script towards the top:

set SDR_TRUNK_OPTS="-Djava.awt.headless=true"

If you are using Unix/Linux, and are running the sdr-trunk startup script you can just set the environment variable first:

export SDR_TRUNK_OPTS="-Djava.awt.headless=true"

I believe it will still play audio when running. You can manage that however you need to using the operating system or settings in SDRTrunk (set all talkgroups listen to off for example).
